pkgs/by-name/wl/wlx-overlay-s/package.nix

```diff
@@ -0,0 +1,90 @@
+{ lib
+, stdenv
+, rustPlatform
+, fetchFromGitHub
+, alsa-lib
+, dbus
+, fontconfig
+, libxkbcommon
+, makeWrapper
+, openvr
+, openxr-loader
+, pipewire
+, pkg-config
+, shaderc
+, wayland
+, xorg
+  # Whether to include programs used in default config
+, withDefaultPrograms ? true
```

Comment:
If this only affects pactl, we can probably include this by default without an option.